spec/features/contacts/user_deletes_a_contact_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+feature 'When a user clicks delete for a contact' do
+  it 'the contact disappears' do
+    contact = create(:contact, content: 'jane doe')
+    visit job_path(contact.job)
+
+    expect(page).to have_content('jane doe')
+    within('#contact'){ click_link 'delete' }
+    expect(page).not_to have_css('jane doe')
```

Comment:
Using CSS to hold this value may make this test a little bit fragile. Seems like you're making decisions about styles to add to your view based on what will make your tests easiest. Aim to avoid that.